tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -54,9 +52,19 @@ def push_metrics(self, metric_name, metric_value):
             int(time.time())
         )
         try:
+            self._connect()
             response = self.graphite_sock.sendall(message)
         except socket.error:
             response = self._resend(message)
+        finally:
+            try:
+                self.graphite_sock.close()
+            except(AttributeError, socket.error) as ex:
+                logger.log(
+                    "error",
+                    NS.get("publisher_id", None),
+                    {'message': "Unable to send data to graphite %s" % ex}
```

Comment:
the msg should be "Unable to close graphite socket"

please use relevant log msgs